Document and export the inputOutline style mixin (#1037)
sumup-oss/circuit-ui: .changeset/cold-beds-sort.md

---
'@sumup/circuit-ui': minor
---

> Added the `inputOutline` style mixin. It can be used to communicates to the user that an element is hovered, focused, or active; including for the disabled, invalid and warning states.

Reviewer: ```suggestion
Added the `inputOutline` style mixin. It can be used to communicate to the user that an element is hovered, focused, or active in the disabled, invalid, and warning states.
```
